bot.py

```diff
@@ -9,7 +9,7 @@
 
 discord_token = os.getenv('DISCORD_TOKEN')
 space_endpoint = os.getenv('SPACE_ENDPOINT')
-channel_id = os.getenv('DISCORD_CHANNEL_ID')
+channel_id = "1299655773252485201"
```

Comment:
Let's revert it back to using the env variable. And let me know what value we should set on the server side.

Reply:
You need first create channel in HS3 server, then copy ID of this channel and paste it into the code.
<img width="385" alt="image" src="https://github.com/user-attachments/assets/2e57afa3-8f41-4e0f-9dfa-7085d17090dc">

Also you need to make sure that glider has permissions to set and change channel name :)
<img width="674" alt="image" src="https://github.com/user-attachments/assets/45faf464-08b2-4d7c-8fb4-d1d08bf24357">

Reply:
you can skip step 2, glider is like an admin on server so by default it can change everything we want
<img width="630" alt="image" src="https://github.com/user-attachments/assets/1fb1de09-cca7-4a1e-8dfb-18760ab07e3a">